### ISSUE TITLE
TypeScript: Enums being formatted weirdly

### ISSUE THREAD

**Issue by prencher** [prettier/prettier#1873]
The TypeScript support is great... Mostly. 😄 

The following:
```ts
export enum BuildType {
  Dev = 1,
  Staging = 2,
  Production = 3,
}
```
Gets rewritten to:
```ts
export enum BuildType { Dev = 1, Staging = 2, Production = 3 }
```
I would expect it to retain the formatting here, including the trailing comma when enabled.

**Comment by vjeux** [comment]
Agreed, we should always expand for enums, the same way we do it for classes.

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!
